src/event.rs

```diff
@@ -95,24 +101,25 @@ pub enum ErrorOrigin {
 ///
 /// For the error type simply choose the default [`GameError`](../error/enum.GameError.html),
 /// or something more generic, if your situation requires it.
-pub trait EventHandler<E = GameError>
+pub trait EventHandler<C, E = GameError>
 where
     E: std::fmt::Debug,
+    C: std::any::Any + HasMut<ContextFields> + HasMut<input::mouse::MouseContext>,
```

Comment:
why the Any bound?

Reply:
Ah that was before I added the other bounds kind of slipped my mind. I'll fix that up